README.md

```diff
@@ -6,16 +6,16 @@ Beetmoverscript README
 
 ## deploy a new version
 
-1. Once PR is reviewed and passes the tests, have one of the admins merge that
+1. Once your PR is reviewed and passes the tests, have one of the admins review & merge it
 2. Bump to new version in `version.json` and commit with a "%VERSION%" message
 3. `git tag -s %VERSION%`
 4. `git push`
 5. `git push --tags`
 6. Create tarball with `python setup.py sdist` and scp that file under [puppet](http://releng-puppet2.srv.releng.scl3.mozilla.com/python/packages-3.5/)
-7. Wait for that file to be synchronized across all puppet instances (emails arrive to nofity of that)
+7. Wait for that file to be synchronized across all puppet instances (emails arrive to confirmat that)
 8. Tweak tweak the `beetmoverscript` version under [beetmoverworker module](https://hg.mozilla.org/build/puppet/file/tip/modules/beetmover_scriptworker/manifests/init.pp#l28) to reflect the new version
```

Comment:
Nit, tip always makes me twitch when we have a prod branch (or a relbranch for gecko). default is probably better here.